Add clearer instructions for programs with server-side decorations (#173)
flathub-infra/documentation: docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

### Include window shadow and rounded corners

> Screenshots must include the native toolkit's (or system's if the platform renders titlebars) titlebar, shadow and rounding. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.

Reviewer: can we somehow not say titlebar twice? :D

Reviewer: And the `'s` feel weird IMO

Author: There, feel that should be a good enough solution given all the feedback until now.

> Screenshots must include the native toolkit shadow and rounding - if the titlebar is drawn by your platform, the titlebar and its shadow must be included. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.

Reviewer: ```suggestion
Screenshots must include the native toolkit shadow and rounding. If the titlebar is drawn by your platform, the titlebar and its shadow must be included. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.
```

> Screenshots must include the native toolkit shadow and rounding. If the titlebar is drawn by your platform instead, the titlebar and its shadow must be included. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.

Reviewer: What do you think about this?

```suggestion
Screenshots must include the native toolkit (or system, if the platform renders titlebars) shadow and rounding. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.
```

Author: I'd say it should still be clear that the titlebars gotta be included, too, unless an example screenshot is added clearly showing a server-side decoration being included.

Maybe "native toolkit (or system if the platform renders titlebars) titlebar, shadow and rounding." might suffice?

Author: Also as an extra idea: Wouldn't it be better to say "toolkit's" and "system's" in this sentence?

Author: Hmm... guess I'll apply it for now and re-review, see what razzeee thinks after the fact.

### Include window shadow and rounded corners

> Screenshots must include the native titlebar, shadow and rounding (if any). App stores do not add a shadow after the fact, and without one, screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.

Reviewer 2: ```suggestion
Screenshots must include the native decoration such as any titlebar, shadow, and rounded corners. App stores do not add a shadow after the fact, and without one, screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.
```

> Screenshots must include the native decoration such as any titlebar, shadow, and rounded corners. App stores do not add a shadow after the fact, and without one, screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.

Reviewer: ```suggestion
Screenshots must include the native decoration such as any titlebar, window shadows, and rounded corners. App stores do not add a shadow after the fact, and without one, screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.
```

Wondering, if shadow alone might confuse people.
